### DISCUSSION TITLE
Where can I see, what nextcloud version AIO is running?

### DISCUSSION THREAD

**Discussion by ninoppp** [nextcloud/all-in-one#640] [Questions] [answered]
Would be important to know if I can migrate from standalone nextcloud installation.

**Comment by szaimen** [comment, marked as answer]
Hello, maybe try `sudo docker inspect nextcloud-aio-nextcloud | grep NEXTCLOUD_VERSION`.

Does that work?

**Reply by ninoppp** [reply]
Ah I think I formulated it wrongly: How can I see, what version the current release is running (before installing/downloading it). Had a look into the .env, it's 23.0.4, right?

**Reply by szaimen** [reply]
> Had a look into the .env, it's 23.0.4, right?

Currently, yes👍
See https://github.com/nextcloud/all-in-one/blob/main/Containers/nextcloud/Dockerfile#L108

**Reply by szaimen** [reply]
I guess the best link is probably this:
https://github.com/nextcloud/all-in-one/search?l=Dockerfile&q=NEXTCLOUD_VERSION&type=
